cmd/cli/serve/util.go

```diff
@@ -47,94 +44,131 @@ func GetComputeConfig(
 		if err != nil {
 			return node.ComputeConfig{}, err
 		}
-		executionStore, err = getExecutionStore(ctx, cfg.Node.Compute.ExecutionStore, executionStoreDBPath)
-		if err != nil {
-			return node.ComputeConfig{}, pkgerrors.Wrapf(err, "failed to create execution store")
-		}
+		executionStore, err = boltdb.NewStore(ctx, executionStoreDBPath)
 	}
 
 	executionsPath, err := cfg.ExecutionDir()
 	if err != nil {
 		return node.ComputeConfig{}, err
 	}
 
-	return node.NewComputeConfigWith(executionsPath, node.ComputeConfigParams{
-		TotalResourceLimits:                   *totalResources,
-		JobResourceLimits:                     *jobResources,
-		DefaultJobResourceLimits:              *defaultResources,
-		IgnorePhysicalResourceLimits:          cfg.Node.Compute.Capacity.IgnorePhysicalResourceLimits,
-		JobNegotiationTimeout:                 time.Duration(cfg.Node.Compute.JobTimeouts.JobNegotiationTimeout),
-		MinJobExecutionTimeout:                time.Duration(cfg.Node.Compute.JobTimeouts.MinJobExecutionTimeout),
-		MaxJobExecutionTimeout:                time.Duration(cfg.Node.Compute.JobTimeouts.MaxJobExecutionTimeout),
-		DefaultJobExecutionTimeout:            time.Duration(cfg.Node.Compute.JobTimeouts.DefaultJobExecutionTimeout),
-		JobExecutionTimeoutClientIDBypassList: cfg.Node.Compute.JobTimeouts.JobExecutionTimeoutClientIDBypassList,
+	params := node.ComputeConfigParams{
+		ExecutionStore: executionStore,
+
+		// TODO(review): what are we setting these fields to now? We have a new set of defaults that are based on the job type.
+		TotalResourceLimits:      *totalResources,
+		JobResourceLimits:        *jobResources,
+		DefaultJobResourceLimits: *defaultResources,
+
+		// TODO(review): assumedly we no longer support this feature?
+		// IgnorePhysicalResourceLimits:          false,
```

Comment:
I agree we shouldn't expose this. part of our configs is we allow the user to define the allocated/total capacity instead of bacalhau try to calculcate it and use 80% of it. Meaning we are achviving the same as `IgnorePhysicalResourceLimits` by defining the allocated capacity values